### PR TITLE
User auth and basic layout

### PR DESCRIPTION
**Pull request by Naman-Saxena1**
# Added user auth and basic site layout
1. User can signup to create account and also login with those credentials
2. Deployed backend for user auth and some other apis
3. Added basic structure of website

Files to review:
1. Sidebar.jsx
2. Sidebar.css
4. Home.jsx
5. Home.css

### CODE REVIEW

**Review by yaswanthmyneni — APPROVED** [review]
The UI is good, you can merge it.

File: src/Components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    position: fixed;
+    top: 10.7rem;
+    overflow-y: scroll;
+    padding: 3rem 3rem 3rem 3rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    gap: 1rem;
+    color: black;
+    box-sizing: border-box;
```
Comment (yaswanthmyneni) [inline src/Components/Sidebar/Sidebar.css:15]: changes: use css variables for colors

File: src/Pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          }
+          else
+          {
+              // (async function getUpdatedUserData()
+              // {
+              //     let updatedUserInfo = await axios.get(
+              //     "http://localhost:1337/api/user",
+              //     {
+              //         headers:
+              //         {
```
Comment (mehulsatardekar) [inline src/Pages/Home/Home.jsx:37]: You can remove the commented code

**Review by mehulsatardekar — APPROVED** [review]
Everything looks good to me you can merge this branch 😃

File: src/Pages/Home/Home.jsx (hunk 2 above)
Comment (Naman-Saxena1) [reply src/Pages/Home/Home.jsx:37]: Yeah, I am planning to use it in the future, if I felt like I wont' use it, I will remove it 😄

**Pull request merged**